lib/ipmi_backend_utils.pm

```diff
@@ -364,7 +341,7 @@ sub set_grub_on_vh {
     }
 
     #set up xen serial console
-    my $ipmi_console = get_dom0_serialdev("$root_dir");
+    my $ipmi_console = get_dom0_serialdev();
```

Comment:
Could you help explain this change here ?
```
my $ipmi_console = get_dom0_serialdev();
```

Reply:
Originally, get_dom0_serialdev() function is to determine the serial console for xen hypervisor based on grub or grub2. So far we have not grub in our products any more.  In addtion, the needle matching makes TW login fail. So I remove the obsolete codes on checking the release to determine which one is in use, grub or grub2.
Do you find grub is still in use in any product? If so, I need reconsider these changes.

Reply:
No, personally I only use grub2. What about "$root_dir" which is also removed.

Reply:
See my changes to get_dom0_serialdev() function just before this change in the PR. "$root_dir" is only used to locate the release file for distingushing grub and grub2. All the stuff is removed.

Reply:
OK